src/main/java/com/lobo/onThisDay/service/OnThisDayScrapperService.java

```diff
@@ -0,0 +1,96 @@
+package com.lobo.onThisDay.service;
+
+import com.lobo.onThisDay.model.Person;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.time.Month;
+import java.time.MonthDay;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * Service que fará os requests e o scrapping da fonte do onthisday.com
+ *
+ * @author Gustavo Lobo
+ */
+@Service
+public class OnThisDayScrapperService {
+
+    // TODO: Talvez alterar a url base para ser especifica para nascimentos, outros eventos podem ter uma url em formato diferente.
+    private static final String BASE_URL = "https://www.onthisday.com/%s/%s/%d";
+    private static final String BIRTH_EVENTS_URL = "birthdays";
+
+    /**
+     * Esse metodo será o responsável por fazer os requests da lista de nascidos na fonte do onthisday, de inicio estamos
+     * buscando apenas as pessoas que estão em sessões de destaque na pagina.
+     * <p>
+     * TODO: Refatorar em pequenos metodos, adicionar validações e criar testes.
+     *
+     * @param date
+     * @return lista de pessoas nascidas em "x" dia.
+     */
+    public List<Person> getHighlightPersonsBornIn(MonthDay date) throws IOException {
```

Comment:
Esse claramente é um método Deus... ele faz tudo, pode tudo e sabe de tudo! 

For exemplo vc tem um for e dentro dele tem outro for toda essa parte do primeiro for pode fica em um método separado, o segundo for pode ir para um que vai ser chamado pelo primeiro e assim sucessivamente.

Sugiro quebrar em vários pequenos métodos especialista e usas esse metodo para juntar os resultados destes outros metodos.